README.md

```diff
@@ -207,12 +207,14 @@ sent to `myLocationConstraint` to the file backend:
 
 Each locationConstraint must include the `type`, `legacyAwsBehavior`,
 and `details` keys. `type` indicates which backend will be used
-for that region. Currently, mem, file, and scality are the
+for that region. Currently, mem, file, scality, and AWS S3 are the
```

Comment:
I think we want to defer updating the readme until mpu and the encryption pr's are merged.  Will you please move these changes there?